packages/projects-docs/pages/learn/environment/vm.mdx

```diff
@@ -38,25 +36,36 @@ Note that, the first time you import a project to CodeSandbox, it can take longe
 
 ## Persistence
 
-Everything you save in `/project` is guaranteed to be persisted between reboots, hibernations and forks. Every file change outside of `/project` will usually be persisted, but there is a chance that those changes will be cleared.
+We use [Dev Containers](https://codesandbox.io/docs/learn/environment/devcontainers) by default for all repositories and Devboxes. Your project is under `/workspaces/workspace`, and your home folder is under `/root`. Under the hood, the paths are in `/project`.
```

Comment:
Maybe we can define workspace as the directory where the code is stored and it defaults to `/workspaces/workspace`. Then we do not have to repeat `/workspaces/workspace` in the next few lines (Those references might confuse users whose workspace is not in `/workspaces/workspace`).